pubsub/include/marlin/pubsub/PubSubNode.hpp

```diff
@@ -598,6 +602,13 @@ void PUBSUBNODETYPE::send_RESPONSE(
 	transport.send(std::move(m));
 }
 
+
+bool check_reward_worthy(uint8_t *buf, size_t capacity) {
+	uint8_t tmp = buf[capacity - capacity]; // Just to use them so no error.
```

Comment:
You can just omit the parameter name (and have just type) to fix the unused variable warnings.